readme.md

```diff
@@ -186,6 +186,7 @@ Made with Electron.
 - [Motrix](https://github.com/agalwood/Motrix) - Download manager.
 - [Franz](https://github.com/meetfranz/franz) - Skype, Slack, Hangouts, WhatsApp, Grape, Telegram, FB Messenger, Hipchat in the same app.
 - [Gmail Desktop](https://github.com/timche/gmail-desktop) - Unofficial Gmail app.
+- [Upcount](https://github.com/madisvain/upcount) - Easy Invocing app
```

Comment:
You have a typo and you forgot the `.`

Reply:
@sindresorhus I'm very sorry about these mistakes and taking your time. I hope this final commit will solve the remaining issues.